### ISSUE TITLE
When using graphql_query with pytest, an error occurs: Deprecated in Pydantic V2.0 to be removed in V3.0.

### ISSUE THREAD

**Issue by half-coconut** [denisart/graphql-query#31]
**Describe the bug**
when I use graphql_query with pytest, here is the error message. What I am confused about is that if graphql_query is used alone, this error will not be reported. Please pay attention to this issue. Thank you.

```
    from graphql_query import Argument, Directive, Field, Operation, Query, Variable
../../../../venv/lib/python3.11/site-packages/graphql_query/__init__.py:5: in <module>
    from .types import Argument, Directive, Field, Fragment, InlineFragment, Operation, Query, Variable
../../../../venv/lib/python3.11/site-packages/graphql_query/types.py:41: in <module>
    class _GraphQL2PythonQuery(PydanticBaseModel):
../../../../venv/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py:90: in __new__
    config_wrapper = ConfigWrapper.for_model(bases, namespace, kwargs)
../../../../venv/lib/python3.11/site-packages/pydantic/_internal/_config.py:120: in for_model
    config_from_namespace = config_dict_from_namespace or prepare_config(config_class_from_namespace)
../../../../venv/lib/python3.11/site-packages/pydantic/_internal/_config.py:271: in prepare_config
    warnings.warn(DEPRECATION_MESSAGE, DeprecationWarning)
E   pydantic.warnings.PydanticDeprecatedSince20: Support for class-based `config` is deprecated, use ConfigDict instead. 
Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.5/migration/
```

![errorMessage](https://github.com/denisart/graphql-query/assets/70947600/0998d03d-195a-42cb-8fb0-d3c22905cf9d)

**Comment by denisart** [comment]
Hello, thank for report of this bug!

I will check this problem soon. Can you give an example of a test with this error?

**Comment by half-coconut** [comment]
```python
import requests
from graphql_query import Argument, Directive, Field, Operation, Query, Variable

dd = 'str1 str2 str3 str4 str5 str6'


class demo:
    def getData(self, sort_field, reverse, data_type, input):
        get_asset = Query(
            name="get_data",
            arguments=[Argument(name="sort_field", value=sort_field),
                       Argument(name="reverse", value=reverse),
                       Argument(name="data_type", value=data_type),
                       Argument(name="input", value=input),
                       ],
            fields=[
                Field(name="color", fields=[dd]),
                Field(name="count")
            ]
        )
        operation = Operation(type="", queries=[get_asset])
        return operation.render()

    def result(self):
        return {
            "query": self.getData(sort_field='"green",', reverse='false,', data_type='"red",',
                                   input='["a","b","c","d","e","f"],')
        }


print(demo().result())
data = demo().result()

your_url = ''
your_header = {}

# method 1
r = requests.post(url=your_url, json=data, headers=your_header)
print(r.status_code)
print(r.json())


# def test_query():
#     # method 2
#     r = requests.post(url=your_url, json=data, headers=your_header)
#     print(r.status_code)
#     print(r.json())

'''
Replace with your URL, header, and parameters
then apply Method 1 and Method 2 separately. 
Method 1 succeeds, while Method 2 results in an error.

Python 3.11.5
pytest 7.4.2
pydantic 2.5.3
pydantic_core 2.14.6
graphql_query 1.2.0
'''
```

**Comment by denisart** [comment]
The following PR fixed your problem https://github.com/denisart/graphql-query/pull/32
Available in version 1.3.0 https://github.com/denisart/graphql-query/releases/tag/v1.3.0
